core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/workflow/impl/IntegerProgressStep.java

```diff
@@ -1,8 +1,14 @@
 package uk.ac.bbsrc.tgac.miso.core.data.workflow.impl;
 
+import javax.persistence.Column;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'javax.persistence.Column'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)